dc-cudami-client/src/main/java/de/digitalcollections/cudami/client/CudamiFileResourcesBinaryClient.java

```diff
@@ -21,7 +21,7 @@
 public class CudamiFileResourcesBinaryClient extends CudamiBaseClient<FileResourceImpl> {
 
   public CudamiFileResourcesBinaryClient(String serverUrl, ObjectMapper mapper) {
-    super(serverUrl, FileResourceImpl.class, mapper);
+    super(null, serverUrl, FileResourceImpl.class, mapper);
```

Comment:
Kann man hier nicht auch den globalen HTTP-Client hernehmen?

Reply:
Der binary upload basiert auf apache http client. Eine Migration ist zu gross für diesen Merge Request (gerne in einem anderen Branch). Extends CudamiBaseClient entfernt, da dieser Client völlig unabhängig davon.